plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeLegacyCreateTableWithExistingLocationUsingSessionProperty.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.Session;
+
+import java.util.Map;
+
+import static io.trino.testing.TestingSession.testSessionBuilder;
+
+public class TestDeltaLakeLegacyCreateTableWithExistingLocationUsingSessionProperty
```

Comment:
This and TestDeltaLakeLegacyCreateTableFailureWithExistingLocation can be merged together.

- setup
- test that you cannot CREATE TABLE with existing location
- enable session property and verify that now you can

Reply:
That, together with the other comment means you can fold `*DeltaLakeLegacyCreateTableWithExistingLocation*` classes into just one, without any inheritance. What a simplification

Reply:
Thanks for the suggestion :) It really simplifies the classes into one.